NSS-Day-Cohort-66/rare-api-eager-miracles
# Filter Posts By User

## Filter Post Feature
- Added if/else block in list method to look for "user" in HTTP request, and if present then only return posts with the specified user.id

## Testing
- Commit any changes on your current local branch
- "git fetch" in terminal to retrieve available remote branches
- "git checkout lh-user-posts" to checkout this feature branch
- Run debugger in API session
- In Postman send a GET request to "http://localhost:8000/posts?user=4", using any existing user_id after "user="
- Verify that only posts with the corresponding user_id are returned in response
- Send another GET request to the base url "http://localhost:8000/posts", verify the default list method still works
